lib/plug/static.ex

```diff
@@ -7,12 +7,12 @@ defmodule Plug.Static do
     * `:at` - the request path to reach for static assets.
       It must be a string.
 
-    * `:from` - the filesystem path to read static assets from.
-      It must be a string, containing a file system path, or an
-      atom representing the application name where assets will
-      be served from the priv/static, or a tuple containing the
-      application name and the directory to serve assets besides
-      priv/static.
+    * `:from` - the file system path to read static assets from.
```

Comment:
I don't think changing this is a clear improvement, I checked Elixir documentation and we don't have a standard set yet 😕 Sadly, apparently there's no "right" way of writing that:

> In computing, a file system (or filesystem) [...] [[Wikipedia]](https://en.wikipedia.org/wiki/File_system)

Wdyt?